From 9baedd86c7aa840e90c92127b04d8d39cba8d8c7 Mon Sep 17 00:00:00 2001
From: Lin Fuzhen <fuzhen.lin@freescale.com>
Date: Mon, 6 May 2013 18:19:14 +0800
Subject: [PATCH 196/602] ENGR00261516 Enable emmulator battery for boards
 without a battery

Add emulated battery support for boards which have no battery in HW
design

Signed-off-by: Lin Fuzhen <fuzhen.lin@freescale.com>
---
 arm2_6dq/init.rc       | 3 +++
 hdmidongle_6dq/init.rc | 3 +++
 sabreauto_6q/init.rc   | 3 +++
 3 files changed, 9 insertions(+)

diff --git a/arm2_6dq/init.rc b/arm2_6dq/init.rc
index ec30d77..e54a593 100755
--- a/arm2_6dq/init.rc
+++ b/arm2_6dq/init.rc
@@ -2,6 +2,9 @@ import init.${ro.hardware}.usb.rc
 
 on boot
 
+	# emmulate battery property
+	setprop sys.emulated.battery 1
+
     # Set permission for IIM node
     symlink /dev/mxs_viim /dev/mxc_mem
 
diff --git a/hdmidongle_6dq/init.rc b/hdmidongle_6dq/init.rc
index 418ba51..a52334a 100755
--- a/hdmidongle_6dq/init.rc
+++ b/hdmidongle_6dq/init.rc
@@ -19,6 +19,9 @@ on init
 
 on boot
 
+	# emmulate battery property
+	setprop sys.emulated.battery 1
+
     # Set permission for IIM node
     symlink /dev/mxs_viim /dev/mxc_mem
 
diff --git a/sabreauto_6q/init.rc b/sabreauto_6q/init.rc
index 41ef46f..da26f1b 100644
--- a/sabreauto_6q/init.rc
+++ b/sabreauto_6q/init.rc
@@ -19,6 +19,9 @@ on init
 
 on boot
 
+	# emmulate battery property
+	setprop sys.emulated.battery 1
+
     # Set permission for IIM node
     symlink /dev/mxs_viim /dev/mxc_mem
 
-- 
1.8.0

